packages/global/package.json

```diff
@@ -26,6 +27,7 @@
     ]
   },
   "dependencies": {
-    "ansi-colors": "^4.1.3"
+    "ansi-colors": "^4.1.3",
+    "lit": "^2.6.1"
```

Comment:
Do not put lit into global? because I want to make it pure

Reply:
lit is installed to use `import { svg } from 'lit';`, as I think export icons in global is good 👀, what do you think

Reply:
Looks fine? I'm not sure whether it has the side effect

Reply:
It seems fine when building, I'll track this further